### PR TITLE
handle burl files

### PR DESCRIPTION
**Pull request by juckerf**
if a burl file in the configured balloon dir is opened (e.g. double clicked) balloon shows a prompt to open the url from the burl file in browser.
the prompt is embedded in the tray window.
linux: since there's no way to get the tray position, the "tray" window for burls is opened at the current position of the cursor. this window disappears on "blur" (mouse clicks outside the window).

### CODE REVIEW

File: app/lib/ipc.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const ipc = require('node-ipc');
```
Comment (pixtron) [inline app/lib/ipc.js]: can this be solved with electrons ipc?

**Comment by raffis** [comment]
Linux: We implement a separate window (Like you had it before) for that which always opens in the center.
Therefore for windows and os x as well. We should implemented a dynamic pre-rendered window prompt.

**Comment by juckerf** [comment]
> Linux: We implement a separate window (Like you had it before) for that which always opens in the center.
> Therefore for windows and os x as well. We should implemented a dynamic pre-rendered window prompt.
implemented electron native dialog

File: app/lib/ipc.js (hunk 1 above)
Comment (juckerf) [reply app/lib/ipc.js]: Ok, even with electron native dialogs IPC is needed (with windows and linux). we start a new balloon process when double-clicking associated files (*.burl) but dialogs can only be shown if the app is ready, so we have to inform the main process to show the dialog

**Comment by raffis** [comment]
@juckerf do you mind pointing this pr to the dev branch, thanks.

**Pull request closed without merging**